### PR TITLE
Corrections

### LINKED ISSUE #306: need to test Endpoint.
[//]: # (Check the boxes corresponding to your situation: [x])
 [//]: # (Replace XX.XX with the version)

I have installed AskOmics with:
- [ ] git
- [ ] docker
- [ ] askomics-docker-compose repository
- [ ] No idea
- [X] VM from image ubuntu18.04_askodev

I Installed virtuoso from:
- [ ] Compiled from sources
- [ ] docker
- [ ] askomics-docker-compose repository
- [x] No idea

The deployment mode of AskOmics is:
- [x] dev
- [ ] prod
- [ ] No idea

The AskOmics version is:
- [x] The master branch
- [ ] A specific version: XX.XX
- [ ] No idea

[//]: # (Describe your problem/suggestion here)

QueryLauncher did not check if the uri endpoint is accessible or even an endpoint.
I'll want to add a method test_endpoint that will send a dummy query and throw error if the endpoint fail to respond.

[//]: # (Post some log if you can have)

### PR DESCRIPTION
**Pull request by sletort**
ParamManager : some method renamed, compatibility maintained.
QueryLauncher: a new method to test Endpoint validity.
     A little refactoring to generalize  to make it able to deal with non Askomics endpoint (future feature).
New test file for QL.
endpoints.hbs: little correction in HTML generated.

fix #306

### CODE REVIEW

**Comment by sletort** [comment]
This is related to #306 .

File: askomics/libaskomics/rdfdb/QueryLauncher.py [hunk 1]
```diff
@@ -359,3 +429,21 @@ def insert_data(self, ttl_string, graph, ttl_header=""):
         res = self._execute_query(query_string)
 
         return res
+# QueryLauncher
+
+# ----------------------------------------
+
+if __name__ == '__main__':
```
Comment (xgaia) [inline askomics/libaskomics/rdfdb/QueryLauncher.py]: Can you remove this lines?

File: askomics/test/QueryLauncher_test.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    def test_test_endpoint__wrong_input( self ):
+        query = generic_query
+        ep_uri = 'http://aqw.com/'
+        o_ql = QueryLauncher_( {},{}, endpoint=ep_uri )
+        self.assertRaises( NotEndpoint, o_ql.test_endpoint )
+
+#QueryLauncherTests
+
+# ================================================
```
Comment (xgaia) [inline askomics/test/QueryLauncher_test.py]: And this too

Comment (sletort) [reply askomics/test/QueryLauncher_test.py]: done.

**Comment by xgaia** [comment]
fix #306

**Pull request merged**